.github/pull_request_template.md

```diff
@@ -20,3 +20,5 @@ Requirement changes:
 - [ ] Tests added
 
 Resolves #<github issues> [JIRA-TAG]
+
+Coverage reports (updated automatically):
```

Comment:
Might want to add a small note here to not delete since the script seems to look for this for inserting coverage.